pkg/envinfo/envinfo_test.go

```diff
@@ -23,6 +23,7 @@ func TestSetEnvInfo(t *testing.T) {
 	os.Setenv(envInfoEnvName, tempEnvFile.Name())
 	testURL := EnvInfoURL{Name: "testURL", ClusterHost: "1.2.3.4.nip.io", TLSSecret: "testTLSSecret"}
 	invalidParam := "invalidParameter"
+	testCreate := ComponentSettings{Type: "componentType", Name: "componentName", Namespace: "namespace"}
```

Comment:
```suggestion
	testCreate := ComponentSettings{Name: "componentName", Namespace: "namespace"}
```

Reply:
Yeah, I already made the changes but they were in my local branch, will commit them with my integration tests.